src/main/java/com/salesforce/emp/connector/LoginHelper.java

```diff
@@ -77,7 +77,7 @@ public void startElement(String uri, String localName, String qName, Attributes
         }
     }
 
-    public static final String COMETD_REPLAY = "/cometd/replay/";
```

Comment:
Yes, as you've said the trailing "replay" is not needed in the new version, and that's why I've removed it. Both of the values (COMETD_REPLAY, and  COMETD_REPLAY_OLD are identical in your master.

I.e. The following line does not makes sense if they are both the same:
`cometdEndpoint = Float.parseFloat(parameters.version()) < 37 ? COMETD_REPLAY_OLD : COMETD_REPLAY;`

For me the issue came up when I've tried executing DevLoginExample against my sandbox. It dies with the exception:
`Exception in thread "main" java.util.concurrent.ExecutionException: java.net.ConnectException: Cannot connect [https://cs60.salesforce.com/cometd/replay/39.0] : {exception=org.cometd.common.TransportException: {httpCode=400}, message={ext={replay=true}, supportedConnectionTypes=[long-polling], channel=/meta/handshake, id=1, version=1.0}, httpCode=400, connectionType=long-polling}
`
Making the above noted changed fixed the issue.